### DISCUSSION TITLE
Cannot Access gocryptfs MOUNTED dir with Apache2.4

### DISCUSSION THREAD

**Discussion by pizzipie** [rfjakob/gocryptfs#725] [Q&A] [answered]
Hello, I am new to this site and gocryptfs so please bear with me.

I am trying to encrypt an "accounts" folder and then access it with apache2.4.
The encryption works fine and if I select the folder it shows me all the encrypted files. Now if I  do: **http://localhost/accts.com/accounts/help.php** the result is:
_
> Forbidden
> 
> You don't have permission to access this resource.
> Apache/2.4.41 (Ubuntu) Server at localhost Port 80
_
The MOUNTED Dir has these permissions:
drwxrwsr-x  7 rick www-data 4096 Apr  2 10:09 accounts/

Any help to make this work will be greatly appreciated.

R

**Comment by lechner** [comment, marked as answer]
Hi @pizzipie,

I haven't tried the group thing, but you may have to [change the FUSE defaults](https://unix.stackexchange.com/a/17423) and provide the [`-allow_other`](https://github.com/rfjakob/gocryptfs/blob/master/Documentation/MANPAGE.md#-allow_other) mount option to Gocryptfs to allow someone to access your decrypted files even though they didn't provide the secret.

Kind regards
Felix Lechner

**Reply by pizzipie** [reply]
Thanks lechner,
I tried that and got it to work!

R
